src/sections/Home/Partners-home/index.js

```diff
@@ -21,7 +21,7 @@ const settings = {
   responsive: [
     {
       breakpoint: 1400,
-      settings: "unslick"
+      settings: "slick"
```

Comment:
@Manish254252 What is the purpose of this change?

Reply:
@Manish254252